src/FSharpx.Collections/DList.fs

```diff
@@ -201,4 +201,21 @@ module DList =
 
     let inline toList l = foldBack (List.cons) l [] 
 
-    let inline toSeq (l:DList<'T>) = l :> seq<'T>
\ No newline at end of file
+    let inline toSeq (l:DList<'T>) = l :> seq<'T>    let rec pairWiseDListData cons dlist lastvalue =
```

Comment:
Looks like there should be a carriage return before `let rec pairwiseDListData ...`

Reply:
Otherwise, this is great! Would love to see more module members flushed out like this!